core/ev-admin.php

```diff
@@ -392,47 +383,49 @@ function trash_deleted_videos( $update_hosts, $new_video_ids ) {
     }
 
     if( isset( $trash_messages ) ) {
+      // All trash messages in one wrap
       $trash_messages = $this->wrap_admin_notice( $trash_messages, 'warning' );
-
-      // return the messages
-      return $trash_messages;
     }
 
-    return '';
+    // return the messages
+    return $trash_messages;
```

Comment:
If you get here, $trash_messages is not set, so this will fail. You need to return the $trash_messages from within the isset() if-statement and return empty here.